src/main/java/org/fungover/storm/httphandler/Response.java

```diff
@@ -0,0 +1,50 @@
+package org.fungover.storm.httphandler;
+
+import java.util.Map;
+
+public class Response {
+    private Map<String, String> headers;
+    private int statusCode;
+    private byte[] body;
+
+    public Response() {
+        this.reset();
```

Comment:
I suggest that **headers** is initialized in the constructor.
```suggestion
        this.reset();
        this.headers = new HashMap<>();
```